GSoC26_H/training/build_gold_set_chunk0.py

```
@@ -0,0 +1,255 @@
+
+            if resp.status_code != 200:
+                raise ValueError(f"HTTP {resp.status_code}")
+
+            choice = resp.json()["choices"][0]["message"]
+            raw = (choice.get("content") or choice.get("reasoning_content") or "").strip()
+            if not raw:
+                raise ValueError("Empty response")
+
+            return parse_answer(raw, valid_uris), raw
+
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)